### PR TITLE
Migrate to new seed-scheduler

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr review please

File: subscriptions/tasks.py [hunk 1]
```diff
@@ -1,16 +1,194 @@
...
+            # start here
+            l.info("Loading Message")
+            message = Message.objects.get(
+                messageset=subscription.messageset,
+                sequence_number=subscription.next_sequence_number,
+                lang=subscription.lang)
+            l.info("Loading Initial Recipient Identity")
+            initial_id = self.get_identity(subscription.identity)
+            if "communicate_through" in initial_id and \
+                    ["communicate_through"] is not None:
```
Comment (gsvr) [inline subscriptions/tasks.py]: `initial_id["communicate_through"] is not None:`

File: subscriptions/tasks.py [hunk 2]
```diff
@@ -1,16 +1,194 @@
...
+                # we should not send messages to this ID. Load the listed one.
+                to_addr = self.get_identity_address(
+                    initial_id["communicate_through"])
+            else:
+                # set recipient data
+                to_addr = self.get_identity_address(subscription.identity)
```
Comment (gsvr) [inline subscriptions/tasks.py]: You're making an extra Api call, but inter-service calls should be fast enough, so should be okay.

File: subscriptions/tasks.py [hunk 3]
```diff
@@ -1,16 +1,194 @@
...
+                        'Content-Type': 'application/json',
+                        'Authorization': 'Token %s' % (
+                            settings.MESSAGE_SENDER_TOKEN,)
+                    }
+                ).json()
+                post_send_process.apply_async(args=[subscription_id])
+                l.info("Message queued for send. ID: <%s>" % str(result["id"]))
+            else:
+                l.info("No valid recipient to_addr found")
+                subscription.subscription.process_status = -1  # Error
```
Comment (gsvr) [inline subscriptions/tasks.py]: Extra `.subscription`?

File: subscriptions/tasks.py [hunk 4]
```diff
@@ -1,16 +1,194 @@
...
+                subscription.completed = True
+                subscription.active = False
+                subscription.process_status = 2  # Completed
+                subscription.save()
+                # If next set defined create new subscription
+                messageset = subscription.messageset
+                if messageset.next_set:
+                    # clone existing minus PK as recommended in
+                    # https://docs.djangoproject.com/en/1.6/topics/db/queries/
+                    # copying-model-instances
```
Comment (gsvr) [inline subscriptions/tasks.py]: https://docs.djangoproject.com/en/1.9/topics/db/queries/#copying-model-instances

**Comment by gsvr** [comment]
@imsickofmaps Made a few comments, :+1: when travis happy after changes.

**Pull request merged**